low-level/files/opt/seagate/sspl/bin/consuldump.py

```diff
@@ -97,6 +97,8 @@ def get_pretty_file_content(self, data):
             return data.decode()
         except TypeError:
             return deserialized.__str__()
+        except KeyError:
+            return data.decode()
```

Comment:
We need generic exception too wrapping around this code or in invoker code, i dont see that happening today. Script may error out otherwise

More exceptions possible
>https://docs.python.org/3/library/pickle.html
The pickle module defines three exceptions:
exception pickle.PickleError
Common base class for the other pickling exceptions. It inherits Exception.
exception pickle.PicklingError
Error raised when an unpicklable object is encountered by Pickler. It inherits PickleError.
exception pickle.UnpicklingError
Error raised when there is a problem unpickling an object, such as a data corruption or a security violation. It inherits PickleError.
**Note that other exceptions may also be raised during unpickling, including (but not necessarily limited to) AttributeError, EOFError, ImportError, and IndexError.**

>https://docs.python.org/3/library/json.html#Exceptions

Reply:
Done